code/22-decay-tree-fitter/ntuple_DTF1.py

```diff
@@ -0,0 +1,39 @@
+from GaudiConf import IOHelper
+from Configurables import DaVinci, DecayTreeTuple
+from DecayTreeTuple.Configuration import *
+
+# Stream and stripping line we want to use
+stream = 'AllStreams'
+line = 'D2hhCompleteEventPromptDst2D2RSLine'
+
+# Create an ntuple to capture D*+ decays from the StrippingLine line
+dtt = DecayTreeTuple('TupleDstToD0pi_D0ToKpi')
+dtt.Inputs = ['/Event/{0}/Phys/{1}/Particles'.format(stream, line)]
+dtt.Decay = '[D*(2010)+ -> ^(D0 -> ^K- ^pi+) ^pi+]CC'
+
+# add a kinematic fitter
+from Configurables import TupleToolDecayTreeFitter,TupleToolDecay
+dtt.addTool(TupleToolDecay, name = 'Dstar')
+dtt.Branches["Dstar"] = "[D*(2010)+ -> (D0 -> K- pi+) pi+]CC" 
```

Comment:
Everything up until this line uses single quotes `'foo'`, then everything from here until [22](https://github.com/lhcb/first-analysis-steps/pull/68/files#diff-af086c181369004ec2b069598ee0e5fdR22) uses double quotes `"bar"`. For consistency (with [lesson 9](https://lhcb.github.io/first-analysis-steps/09-minimal-dv-job.html) for example) I'd use single quotes.